### ISSUE TITLE
ReferenceError: build is not defined

### ISSUE THREAD

**Issue by VincentPeters** [yeoman/generator-webapp#750]
On a fresh setup, with latest version of generator-webapp package.

```
npm run start

> @ start /Users/vincent/Documents/_projects/frederikvandenbril.be
> gulp serve

/Users/vincent/Documents/_projects/frederikvandenbril.be/gulpfile.js:183
exports.default = build;
                  ^

ReferenceError: build is not defined
    at Object.<anonymous> (/Users/vincent/Documents/_projects/frederikvandenbril.be/gulpfile.js:183:19)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at execute (/Users/vincent/Documents/_projects/frederikvandenbril.be/node_modules/gulp-cli/lib/versioned/^4.0.0/index.js:36:18)
    at Liftoff.handleArguments (/Users/vincent/Documents/_projects/frederikvandenbril.be/node_modules/gulp-cli/index.js:175:63)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ start: `gulp serve`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/vincent/.npm/_logs/2019-01-16T11_18_50_656Z-debug.log
```

**Comment by silvenon** [comment]
Fixed and released in 4.0.0-2!

**Comment by VincentPeters** [comment]
Yep, working now. Thanks!
